src/pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/src/Microsoft.Net.UWPCoreRuntimeSdk.depproj

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <!-- we intentionally don't want to produce output -->
+    <OutputPath>unused</OutputPath>
+    <!-- we don't want any analyzers by ResolveNuGetPackageAssets 
+         null-refs when this isn't set and an analyzer is in the packages -->
+    <Language>unused</Language>
+    <NuGetTargetMoniker>UAP,Version=v10.1</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort>uap10.1</NuGetTargetMonikerShort>
+    <PackageTargetFramework>uap10.1</PackageTargetFramework>
+    <ContainsPackageReferences>false</ContainsPackageReferences>
+    <RidSpecificAssets Condition="'$(NuGetRuntimeIdentifier)' != ''">true</RidSpecificAssets>
+    <IntermediateOutputPath>$(IntermediateOutputPath)$(NuGetRuntimeIdentifier)</IntermediateOutputPath>
+    <RestoreOutputPath>$(IntermediateOutputPath)</RestoreOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Microsoft.Net.CoreRuntime\Microsoft.Net.CoreRuntime.depproj" />
+  </ItemGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <Target Name="GetFilesToPackage" AfterTargets="Build" Returns="@(FilesToPackage)">
+    <ItemGroup Condition="'$(NuGetRuntimeIdentifier)' != ''">
+      <!-- RID-specific: include all runtime files -->
+      <FilesToPackage Include="$(UWPOutputDir)\copywin32resources.exe">
+        <TargetPath>tools/CoreRuntime</TargetPath>
+      </FilesToPackage>
+      <FilesToPackage Include="$(UWPOutputDir)\uwpshim.exe">
+        <TargetPath>tools/AppLocal</TargetPath>
+      </FilesToPackage>
+      <FilesToPackage Include="$(UWPOutputDir)\Microsoft.NET.CoreRuntime.$(MajorVersion).$(MinorVersion).appx">
```

Comment:
Your builds seem to be failing because this appx file doesn't exist https://ci.dot.net/job/dotnet_core-setup/job/master/job/release_windows_nt_x64_prtest/1982/, who builds this appx?